client/src/app/components/managing-body-list/managing-body-list.component.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Created by amanecer on 16/05/2016.
+ */
+import {Component,Input,Output,EventEmitter} from '@angular/core';
+import {ManagingBodyComponent} from './managing-body/managing-body.component';
+
+
+@Component({
+    selector: 'op-managing-body-list',
+    templateUrl: 'app/components/managing-body-list/managing-body-list.component.html',
+    styleUrls: ['app/components/managing-body-list/managing-body-list.component.css'],
```

Comment:
@amanecer2 
Since the `css` file is empty please remove it, and leave the _styleUrls_ array empty.